models/search_result.coffee

```diff
@@ -38,6 +38,8 @@ module.exports = class SearchResult extends Backbone.Model
       "/show/#{@get('id')}"
     else if @get('model') is 'sale'
       "/auction/#{@get('id')}"
+    else if @get('model') is 'city'
+      "/shows/#{@get('id')}"
```

Comment:
Sounds good! Feel free to merge when the Gravity PR is deployed. Thanks, @cavvia!!